pkg/controller/interconnect/interconnect_controller.go

```diff
@@ -643,13 +633,35 @@ func ensureCAIssuer(r *ReconcileInterconnect, instance *v1alpha1.Interconnect, n
 	return nil
 }
 
-// getPodNames returns the pod names of the array of pods passed in
-func getPodNames(pods []corev1.Pod) []string {
-	var podNames []string
-	for _, pod := range pods {
-		if pod.GetObjectMeta().GetDeletionTimestamp() == nil {
-			podNames = append(podNames, pod.Name)
+func getPodStatus(r *ReconcileInterconnect, instance *v1alpha1.Interconnect) v1alpha1.InterconnectPodStatus {
```

Comment:
@myeung18 I've moved this logic to the shared library now, please refactor to use it as done in [corresponding PR](https://github.com/kiegroup/kie-cloud-operator/pull/211)

Reply:
refactor is done